portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,75 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+async function showServerTime() {
+    const responseFromServer = await fetch('/date');
+    const textFromResponse = await responseFromServer.text();
+  
+    const dateContainer = document.getElementById('date-container');
+    dateContainer.innerText = textFromResponse;
+  }
+
+  function loadTasks() {
+    fetch('/list-tasks').then(response => response.json()).then((tasks) => {
+      const taskListElement = document.getElementById('task-list');
+      tasks.forEach((task) => {
+        taskListElement.appendChild(createTaskElement(task));
+      })
+    });
+  }
+  
+  /** Creates an element that represents a task, including its delete button. */
+  function createTaskElement(task) {
+    const taskElement = document.createElement('li');
+    taskElement.className = 'task';
+  
+    const titleElement = document.createElement('span');
+    titleElement.innerText = task.title;
+  
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'Delete';
+    deleteButtonElement.addEventListener('click', () => {
+      deleteTask(task);
+  
+      // Remove the task from the DOM.
+      taskElement.remove();
+    });
+  
+    taskElement.appendChild(titleElement);
+    taskElement.appendChild(deleteButtonElement);
+    return taskElement;
+  }
+  
+  /** Tells the server to delete the task. */
+  function deleteTask(task) {
+    const params = new URLSearchParams();
+    params.append('id', task.id);
+    fetch('/delete-task', {method: 'POST', body: params});
+  }
+
+
+  
+  async function deleteImg() {
```

Comment:
I don't think you are using this method, since the button calls the servlet directly, please remove